src/main/java/seedu/moneygowhere/userinterface/ConsoleInterface.java

```diff
@@ -278,23 +307,27 @@ public static String convertRecurringPaymentToConsoleString(RecurringPayment rec
         if (recurringPayment.getCategory() != null) {
             recurringPaymentStr += "Category        : " + recurringPayment.getCategory() + "\n";
         }
-        recurringPaymentStr += "Currency        : " + recurringPayment.getCurrency();
+        recurringPaymentStr += "Currency        : " + recurringPayment.getCurrency() + "\n";
+        recurringPaymentStr += "Mode of Payment : " + recurringPayment.getModeOfPayment();
```

Comment:
As mode of payment could be `null`, could we standardise it and make it such that it will not print when `null`?

Reply:
Sure, I'll make the change

Reply:
👍